Dialog/GraphicManagerDialog.cpp

```diff
@@ -0,0 +1,1741 @@
+#include "GraphicManagerDialog.h"
+#include "ui_GraphicManagerDialog.h"
+
+#include <QMessageBox>
+#include <QModelIndex>
+
+#include "WL4EditorWindow.h"
+#include "WL4Constants.h"
+#include "FileIOUtils.h"
+
+extern WL4EditorWindow *singleton;
+
+// constexpr declarations for the initializers in the header
+constexpr const char *GraphicManagerDialog::ScatteredGraphicTileDataTypeNameData[1];
+constexpr const char *GraphicManagerDialog::ScatteredGraphicMappingDataCompressionTypeNameData[2];
+
+// static variables used by CameraControlDockWidget
+static QStringList GraphicTileDataTypeName;
+static QStringList GraphicMappingDataCompressionTypeName;
+
+/// <summary>
+/// Construct an instance of the GraphicManagerDialog.
+/// </summary>
+/// <param name="parent">
+/// The parent QWidget.
+/// </param>
+GraphicManagerDialog::GraphicManagerDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::GraphicManagerDialog)
+{
+    // Setup GUI
+    ui->setupUi(this);
+    ui->comboBox_tileDataType->addItems(GraphicTileDataTypeName);
+    ui->comboBox_mappingDataType->addItems(GraphicMappingDataCompressionTypeName);
+    ui->graphicsView_tile8x8setData->scale(2, 2);
+    ui->graphicsView_palettes->scale(2, 2);
+    ui->graphicsView_mappingGraphic->scale(1, 1);
+
+    if (graphicEntries.size())
+    {
+        graphicEntries.clear();
+    }
+    graphicEntries = ScatteredGraphicUtils::GetScatteredGraphicsFromROM();
```

Comment:
Can we change all instances of "scattered" to "assorted"

Reply:
done